charts/mentoring/values.yaml

```diff
@@ -0,0 +1,69 @@
+# Default values for mentoring.
```

Comment:
Do we override some of these values when deploying? If so, how?

Reply:
Yes we will be overriding these values based on the deployment environment (in our case we will only have 1 environment: prod).
The overrides will happen in the HelmRelease object. A Helm Release is like the instance of a Helm Chart.
It is what FluxCD is pointed to.

See [here](https://github.com/mozilla-it/voice-infra/blob/main/kubernetes/releases/voice-dev/voice-dev-chart.yaml) for an example of a HelmRelease object which overrides defaults.
That object override many values because the Voice Chart is used by our contributors, so we want to keep the chart simple and do "many" overrides in our side.